test-suite/generated-src/cpp/enum_usage_record.cpp

```diff
@@ -0,0 +1,9 @@
+// AUTOGENERATED FILE - DO NOT MODIFY!
```

Comment:
These don't seem necessary to generate in C++ as a result of the addition of deriving(parcelable) which only affects Java.  There's probably some logic in the C++ generator which will need to be tweaked to look for specific deriving flags, not just for any deriving flag.